recipes/jina/meta.yaml

```diff
@@ -0,0 +1,132 @@
+{% set name = "jina" %}
+{% set version = "2.0.20" %}
+
+package:
+  name: {{ name|lower }}-split
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1647830ba51187ce569cd76751c7692550e5cf5521fbb4bb93cb1bfaaad3e4cc
+
+build:
+  number: 0
+  skip: true  # [win]
+
+outputs:
+  - name: jina-core
+    build:
+      entry_points:
+        - jina = cli:main
+        - jinad = daemon:main
+      script: python -m pip install . --no-deps -vv
```

Comment:
can this be `noarch`?

Reply:
I think it could be. Since I have several outputs, should I put it under each one separately, or under top-level build?

Reply:
Also just fyi: If the only reason for arch is skipping win you can just add the `__unix ` [virtual package](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-virtual.html) and keep it noarch.

Reply:
I think under each seperately is the right thing to do.

Reply:
@BastianZim I added `__unix` to `host` requirements, but now the Windows build fails because of that (doesn't get skipped). Any ideas what I should do about this?

Reply:
That‘s fine and can be ignored. Once merged, noarch packages will only be built on Linux.